src/config/texts.js

```diff
@@ -442,7 +441,8 @@ export const texts = {
       email: 'E-Mail',
       phone: 'Telefon',
       message: 'Ihre Mitteilung',
-      checkbox: 'Ich bin mit dem Speichern meiner Daten einverstanden.'
+      checkbox: 'Ich bin mit dem Speichern meiner Daten einverstanden.',
+      requiredFields: '* Pflichtfelder'
```

Comment:
How about alphabetical sorting?

Reply:
i do not wanted to mess around here with just adding one new key. and the new key is sorted correctly :)